cmd/sops/edit.go

```diff
@@ -307,3 +306,14 @@ func runEditor(path string) error {
 	cmd.Stderr = os.Stderr
 	return cmd.Run()
 }
+
+func lookupAnyEditor(editorNames ...string) (editorPath string, err error) {
+	for _, editorName := range editorNames {
+		editorPath, err = exec.LookPath(editorName)
+		if err == nil {
+			return editorPath, nil
+		}
+	}
+	err = fmt.Errorf("no editor available: sops supports %s", strings.Join(editorNames, ", "))
```

Comment:
We can improve this error message a bit, something like:

```suggestion
	return "", fmt.Errorf("no editor available: sops attempts to use the editor defined in the EDITOR environment variable, and if that's not set defaults to any of %s, but none of them could be found", strings.Join(editorNames, ", "))
```